test/stresstest/config.py

```diff
@@ -0,0 +1,12 @@
+import configparser
+import pathlib
+
+def getConfig():
+
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path("config.ini").resolve()
```

Comment:
This means you expect "config.ini" to be in the current working directory. Is that what you intended?
Basically, I can't do `python3 test/stresstest/stresstest.py` otherwise I get:
```python
$ python test/stresstest/stresstest.py 
Traceback (most recent call last):
  File "/home/ler2so/Repo/kuksa.val/test/stresstest/stresstest.py", line 44, in <module>
    main()
  File "/home/ler2so/Repo/kuksa.val/test/stresstest/stresstest.py", line 38, in main
    cfg = config.getConfig()
  File "/home/ler2so/Repo/kuksa.val/test/stresstest/config.py", line 10, in getConfig
    vsscfg = config['vss']      #get data from config file
  File "/usr/lib/python3.10/configparser.py", line 964, in __getitem__
    raise KeyError(key)
KeyError: 'vss'
```